Jenkinsfile

```diff
@@ -534,11 +534,25 @@ pipeline{
             stage('dh5-example'){
                  agent { label 'dhfLinuxAgent'}
                 steps{
+                     sh 'cd $WORKSPACE/data-hub/examples/dh-5-example;repo="    maven {url \'http://distro.marklogic.com/nexus/repository/maven-snapshots/\'}";sed -i "/repositories {/a$repo" build.gradle; '
                      copyRPM 'Release','10.0-3'
                      script{
                         props = readProperties file:'data-hub/pipeline.properties';
                         def dockerhost=setupMLDockerCluster 3
-                        sh 'docker exec -u builder -i '+dockerhost+' /bin/sh -c "export JAVA_HOME=`eval echo "$JAVA_HOME_DIR"`;export GRADLE_USER_HOME=$WORKSPACE$GRADLE_DIR;export M2_HOME=$MAVEN_HOME/bin;export PATH=$GRADLE_USER_HOME:$PATH:$MAVEN_HOME/bin;cd $WORKSPACE/data-hub/examples/dh-5-example;rm -rf $GRADLE_USER_HOME/caches;./gradlew clean;set +e;./gradlew -i hubInit -Ptesting=true;./gradlew -i mlDeploy -Ptesting=true;./gradlew hubRunFlow -PflowName=ingestion_only-flow -Ptesting=true;./gradlew hubRunFlow -PflowName=ingestion_mapping-flow -Ptesting=true;./gradlew hubRunFlow -PflowName=ingestion_mapping_mastering-flow -Ptesting=true;"'
+                        sh '''
+                            docker exec -u builder -i '''+dockerhost+''' /bin/sh -c "export JAVA_HOME=`eval echo "$JAVA_HOME_DIR"`;\
+                            export GRADLE_USER_HOME=$WORKSPACE$GRADLE_DIR; \
+                            export M2_HOME=$MAVEN_HOME/bin; \
+                            export PATH=$GRADLE_USER_HOME:$PATH:$MAVEN_HOME/bin; \
+                            cd $WORKSPACE/data-hub/examples/dh-5-example; \
+                            rm -rf $GRADLE_USER_HOME/caches; \
+                            ./gradlew -i hubInit -Ptesting=true; \
+                            ./gradlew -i mlDeploy -Ptesting=true -PmlUsername=admin -PmlPassword=admin; \
+                            ./gradlew hubRunFlow -PflowName=ingestion_only-flow -Ptesting=true -PmlUsername=admin -PmlPassword=admin; \
+                            ./gradlew hubRunFlow -PflowName=ingestion_mapping-flow -Ptesting=true -PmlUsername=admin -PmlPassword=admin; \
+                            ./gradlew hubRunFlow -PflowName=ingestion_mapping_mastering-flow -Ptesting=true -PmlUsername=admin -PmlPassword=admin;
+                            "
```

Comment:
Can we use anything other than admin/admin to run the flow.

Reply:
how can I do that? Instead of editing the gradle properties file. I'm passing the params.

Reply:
We can maintain a copy of gradle.properties to be used across examples. Paste that over to the project directory after hubinit, then mlDeploy as admin by passing credentials via params and run the flow, so mlUsername credentials in the gradle props file will be used.

Reply:
Copying gradle.properties file from marklogic-datahub and running the flows as flow developer.